record_api/core.py

```diff
@@ -295,6 +298,8 @@ class Stack:
     NULL: ClassVar[object] = object()
     current_i: int = dataclasses.field(init=False, default=0)
     opcode: int = dataclasses.field(init=False)
+    previous_stack: Optional[Stack] = None
```

Comment:
We should probably delete the previous_stack of the previous_stack, in the `__post_init__` function? Just thinking about for GC purposes...

Reply:
Makes sense! :+1: